spec/integration/food_index_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+
+RSpec.describe 'Food index page', type: :feature do
+  before :each do
+    @user1 = User.create!(name: 'Albert Antwi', email: 'albertkantwi@gmail.com', password: '123456')
+
+    visit '/users/sign_in'
+    fill_in 'Email', with: 'albertkantwi@gmail.com'
+    fill_in 'Password', with: '123456'
+    click_button 'Log in'
+
+    # Create some sample foods for the user
+    @food1 = @user1.foods.create(food: 'Apple', measurement_unit: 'kg', price: 15)
+    @food2 = @user1.foods.create(food: 'Banana', measurement_unit: 'each', price: 10)
+
+    visit '/foods'
+  end
+
+  it 'should have the greetings' do
+    expect(page).to have_content('Food List')
+    expect(page).to have_link('Foods')
+    expect(page).to have_link('Recipes')
```

Comment:
- [ ] Kindly fix some failing tests, as tests help ensure that your code works as expected

![image](https://github.com/firepicaso/ror-recipe-app/assets/81932478/74ef51d3-dedf-4a42-bb0d-016ad9066936)
![image](https://github.com/firepicaso/ror-recipe-app/assets/81932478/bdecc304-bb63-4503-855d-b2e1fc9c6274)